apps/foundation-demo/src/app/core/sidebar/containers/sidebar/sidebar.component.html

```diff
@@ -1,7 +1,9 @@
 <daff-sidebar class="sidebar" (escapePressed)="onClose()">
-  <span class="sidebar__close" (click)="onClose()"><i class="fas fa-times"></i></span>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">women's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">men's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">accessories</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">sale</a>
+  <span class="sidebar__close" (click)="onClose()"><i class="fa fa-times"></i></span>
+  <ul class="sidebar__list">
```

Comment:
This should have a daff-list attribute:
```
<ul daff-list class="sidebar__list">
```

Reply:
`I stuck with the sidebar list because it has different styles than daff-list.`